client/components/settings/alert.js

```diff
@@ -40,39 +40,45 @@ export default class AlertItem extends React.Component {
-                <td>{ account.title }</td>
-                <td>
-                    <div className="form-inline">
-                        <span>{ this.props.sendIfText }&nbsp;</span>
-
-                        <select className="form-control"
-                          defaultValue={ alert.order }
-                          ref="select"
-                          onChange={ this.handleSelect }>
-                            <option value="gt">
-                                { $t('client.settings.emails.greater_than') }
-                            </option>
-                            <option value="lt">
-                                { $t('client.settings.emails.less_than') }
-                            </option>
-                        </select>
-
-                        <span>&nbsp;</span>
+                <td className="col-md-2">{ account.title }</td>
+                <td className="col-md-3">
+                    <span style={ { paddingRight: '10' } }>
+                        { this.props.sendIfText }
+                    </span>
+                </td>
+                <td className="col-md-4">
+                    <div className="form-inline pull-right">
+                        <div className="form-group">
+                            <select className="form-control"
+                              defaultValue={ alert.order }
+                              ref="select"
+                              onChange={ this.handleSelect }>
+                                <option value="gt">
+                                    { $t('client.settings.emails.greater_than') }
+                                </option>
+                                <option value="lt">
+                                    { $t('client.settings.emails.less_than') }
+                                </option>
+                            </select>
+                        </div>
 
-                        <input type="number"
-                          ref="limit"
-                          className="form-control"
-                          defaultValue={ alert.limit }
-                          onChange={ this.handleChangeLimit }
-                        />
+                        <div className="input-group input-group-money">
+                            <input type="number"
+                              ref="limit"
+                              className="form-control"
+                              defaultValue={ alert.limit }
+                              onChange={ this.handleChangeLimit }
+                            />
+                            <span className="input-group-addon">€</span>
+                        </div>
                     </div>
                 </td>
-                <td>
-                    <button type="button" className="btn btn-danger pull-right" aria-label="remove"
-                      data-toggle="modal" data-target={ `#confirmDeleteAlert${ alert.id}` }
+                <td className="col-md-1">
```

Comment:
Unless I'm missing something, the total sum of cols is 10 here, although bootstrap considers 12 is the total (kinda remember because it can be easily divided by 2, 3, 4, 6). Not a big deal here, but the sizes could be changed to take advantage of the entire space.